CONTRIBUTING.md

```diff
@@ -2,9 +2,49 @@
 
 Thanks for contributing to libphonenumber!
 
-Please read the following before filing an issue. **Note that we are _not_
-accepting pull requests for validation or formatting metadata updates at this
-time.**
+Please read the following before filing an issue or sending a pull request.
+
+We hope these guidelines will enhance your experience as a contributor to our
+library and know that we appreciate the time you put into making it better.
+
+## Pull requests
+
+We do not accept pull requests for validation, formatting, or timezone metadata
+updates.
+
+For changes specific to windows builds, see [Filing a code
+issue](#filing-a-code-issue) and make sure you have found a reviewer and tester
+before sending the pull request.
+
+We are happy to review and accept pull requests for the following:
+
+*   Carrier updates
+*   Geocoding updates
+*   Bug fixes
+*   API changes
```

Comment:
Yeah I removed from here. I just thought we had accepted such things recently?
Should I also exclude above? I think there may be cases where e.g. someone emails us asking if a function can be public and we think a variant of it could and ask for a PR.